cookiecutter.json

```diff
@@ -0,0 +1,5 @@
+{
+  "project_name": "Project Name",
+  "company_name": "thoughtbot",
+  "bundle_identifier": "com.{{ cookiecutter.company_name | replace(' ', '-') | lower }}"
```

Comment:
should we also be doing this for project name sometimes? I think we need a bundle name as well since spaces won't be allowed in many of the places we have `cookiecutter.project_name`